### ISSUE TITLE
How can I pass the rules when using arrow function?

### ISSUE THREAD

**Issue by ChinTeo** [yannickcr/eslint-plugin-react#1339]
I want to use arrow function like this:
```
class App extends Component {
 
    ...

  handleChange = (value) => {
    this.setState({
      remaining: value
    })
  }

  render(){
    ...
  }
}

```
but it can't pass the lint.

Is there any solution?

**Comment by ljharb** [comment]
That's a bad idea, as it creates N functions when you have N instances.

This is how you should do it:
```jsx
class App extends Component {
 
  // option 1
  constructor(...args) {
    super(...args);
    this.handleChange = this.handleChange.bind(this);
  }
  // or option 2
  handleChange = this.handleChange.bind(this);

  handleChange(value) {
    this.setState({
      remaining: value,
    });
  }

  render() {
    ...
  }
}
```
